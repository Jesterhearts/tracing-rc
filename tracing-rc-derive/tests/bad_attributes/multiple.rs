use tracing_rc_derive::Trace;

#[derive(Trace)]
struct Fails {
    #[trace]
    #[trace]
    item: i64,
}

fn main() {}
