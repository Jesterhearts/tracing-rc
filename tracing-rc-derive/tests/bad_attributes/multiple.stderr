error: Cannot specify multiple `#[trace]` attributes.
 --> tests/bad_attributes/multiple.rs:6:5
  |
6 |     #[trace]
  |     ^
